examples/tabular/distill/example_distill_binary.py

```diff
@@ -1,45 +1,43 @@
 """ Example: distilling AutoGluon's ensemble-predictor into a single model for binary classification. """
```

Comment:
If removing the regression / multiclass examples, I'd add sentence: "Distillation can be done in a similar manner for multiclass classification and regression problems."

Reply:
I removed the other examples cause it appeared to be copy/pasted code mostly, and I don't think the usage changes between them. Will add the sentence.